torchvision/transforms/functional.py

```diff
@@ -28,6 +28,19 @@ def _is_pil_image(img):
         return isinstance(img, Image.Image)
 
 
+def _pil_num_bands(img):
```

Comment:
Since the number of bands within the image is also used in `to_tensor`, I created a function for this.